README.md

```diff
@@ -84,24 +84,23 @@ For **ARM Devices** (such as Raspberry Pi) things might be a bit different. If y
 1. From that recovery image, you will need the Widevine files located in /opt/google/chrome/libwidevinecdm*.so.
 1. These files need to be copied to the `<kodi-profile>/cdm` folder.
 
-_NOTE: Keep in mind that you might need to try multiple recovery images before you find a working one._ 
+_Note: Keep in mind you might need to try multiple recovery images before you find a working one._ 
 
 # Acknowledgement #
-The first idea for Retrospect/XBMC Online TV/XOT-Uzg came from a script by
-BaKMaN.
+The first idea for Retrospect/XBMC Online TV/XOT-Uzg came from a script by BaKMaN.
 
 # Copyrights and Licenses #
-Retrospect is licensed under a _Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International_, see: [LICENSE.md](./LICENSE.md)
+Retrospect is licensed _Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International_, see: [LICENSE.md](./LICENSE.md)
```

Comment:
It really should be `licensed under a`. Then even use that on their own website (https://creativecommons.org/licenses/)

Reply:
They use Google Analytics on their site. Just about everything about CC happens to be sub-par.

Reply:
This will change to (A)GPL3 soon.

Reply:
Great. Will update it once that happens.